ui/src/app/applications/components/application-pod-view/pod-view.scss

```diff
@@ -147,7 +147,7 @@ $num-stats: 2;
             }
             &__star-icon {
```

Comment:
change className to `new-pod-icon` (or whatever you deem appropriate here) 

Can you also make the size of the blue circle smaller? Maybe something closer to this?
```
pod-view__node__pod__star-icon {
    background: none;
    color: #0DADEA ;
    display: block;
    left: 22px;
    margin: 0px;
    position: absolute;
    top: -4px;
    font-size: 10px;
}
```
(the blue color I changed it to is the same one as the ellipsis, I just thought it should be a little darker than the progressing color to distinguish it a little more from the green, they kind of blend together)
![new-pod-icon](https://user-images.githubusercontent.com/50851526/168139886-d4c5da34-9bb9-4ec3-b963-04ab54adc51d.png)

Reply:
Thanks @reginapizza , made these changes!!